### PR TITLE
docs: simulate focus ring

### PR DESCRIPTION
**Pull request by nitayneeman**
This PR simulates the viewer's focus-ring by adding the relevant style globally.

Basically it does what TB does, but the following selector sounds redundant for the StoryBook environment:
`#SITE_CONTAINER.focus-ring-active :not(.has-custom-focus):not(.ignore-focus):focus, #SITE_CONTAINER.focus-ring-active :not(.has-custom-focus):not(.ignore-focus):focus ~ .wixSdkShowFocusOnSibling': {`

It also affects Storybook elements, but I don't think it's an issue. We can restrict it somehow to the component preview if it bothers...

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_487/

**Review by jonathanadler — APPROVED** [review]
Are we sure we don't need the `!important` and `z-index` that the viewer uses?

**Comment by nitayneeman** [comment]
> Are we sure we don't need the `!important` and `z-index` that the viewer uses?

I haven't managed to find the case it's relevant - but we can add them to be completely sure.
I'll add them with a comment above saying we simulate viewer behavior.

File: .storybook/addons/FocusRingToggle/register.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  return (
+    <div
+      style={{
+        margin: 'auto 15px',
+        cursor: 'pointer',
+        userSelect: 'none',
+        padding: '3px 8px',
+        backgroundColor: active ? '#4EB7F5' : '#eee',
+        color: active ? '#444' : '#999',
```
Comment (nitayneeman) [inline .storybook/addons/FocusRingToggle/register.js:19]: I took the same styles from WSR, but we can style differently later if we want

File: .storybook/addons/FocusRingToggle/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const FocusRingToggle = makeDecorator({
+  name: ADDON_TITLE,
+  parameterName: ADDON_TITLE,
+  allowDeprecatedUsage: true,
+  wrapper: (getStory) => {
+    const [active, setActive] = useState(false);
+    const channel = addons.getChannel();
+
+    channel.on(ADDON_ID + '/change', (params) => setActive(params.active));
```
Comment (nitayneeman) [inline .storybook/addons/FocusRingToggle/index.js:15]: The problem might be here.

The events are emitted asynchronously and this, I think, makes the component to render the initial active value even after that the event is emitted.

**Review by jonathanadler — APPROVED** [review]
Very nice!

**Pull request merged**